recipe/thirdpartyemailpassword/emaildelivery/smtpService/main.go

```diff
@@ -0,0 +1,43 @@
+package smtpService
+
+import (
+	"errors"
+
+	"github.com/supertokens/supertokens-golang/ingredients/emaildelivery"
+	epsmtpService "github.com/supertokens/supertokens-golang/recipe/emailpassword/emaildelivery/smtpService"
+	evsmtpService "github.com/supertokens/supertokens-golang/recipe/emailverification/emaildelivery/smtpService"
+	"github.com/supertokens/supertokens-golang/supertokens"
+)
+
```

Comment:
You do not need to do all this here since the email password smpt service has the exact same types of emails as this one. You can just make an instance of that and use that in the sendEmail function